tests/core/test_bridge.py

```diff
@@ -3,18 +3,21 @@
 
 import pytest
 
-if pytest.PLATFORM == "win32":
-    pytest.skip("Unsupported Platform")
-
-pytest.importorskip("multiprocessing")
-
 
 from os import getpid
+from time import sleep
 
 
 from circuits import ipc, Component, Event
 
 
+if pytest.PLATFORM == "win32":
+    pytest.skip("Unsupported Platform")
```

Comment:
this needs to be `pytestmark = pytest.mark.skip("Unsupported Platform")` on newer pytest versions.